### DISCUSSION TITLE
嘿龙猫~联机去哪了~？

### DISCUSSION THREAD

**Discussion by iamkezo1** [LTCatt/PCL2Help#338] [General]
**嘿龙猫~ 联机 去哪了~？**

**Comment by WorldHim** [comment]
> “关于联机，希望大家先，呃，放低期待……” ——龙腾猫跃

BakaXL 现已于 2023 年 10 月 8 日将其部分服务与 Octo 章鱼网络进行对接，并对其可用性进行初步评估，待评估完成后，Octo 网络会全面对其他启动器开放。

**Comment by z0z0r4** [comment]
小丑

**Reply by WorldHim** [reply]
好骂
